dts/bindings/video/nxp,imx-csi.yaml

```diff
@@ -19,3 +19,7 @@ properties:
     type: phandle
     description: the connected source device,
       e.g., a mipi csi or a camera sensor
+
+child-binding:
+  child-binding:
+    include: video-interfaces.yaml
```

Comment:
Yes, changing to the new video interfaces bindings requires changes in several platform .dts, test cases which is difficult to cover all at once, so that's why I said, it's better to let the owners do it but I don't know how to inform people about it.

Reply:
Would opening an issue help? One to keep track of the propagation of this new APIs through the various video drivers, and flagging the appropriate teams.

Reply:
yes, will do.

Reply:
I hope it matches what you had in mind... I opened this:
- https://github.com/zephyrproject-rtos/zephyr/pull/76475

Reply:
Thank you very much for creating the issue :-). I think you've mistaken the [issue link](https://github.com/zephyrproject-rtos/zephyr/issues/80514) though. I made some modifications on the issue description.

Reply:
Oops! Yes... Thank you. I created it with the "bug" template but I agree.